Tools-Override/tests.targets

```diff
@@ -262,4 +262,6 @@
   <Target Name="BuildAndTest" DependsOnTargets="Build;Test" />
   <Target Name="RebuildAndTest" DependsOnTargets="Rebuild;Test" />
 
+  <!-- This helps collect crash dumps and requires python installed -->
```

Comment:
Unless you had to make changes to them. In which case please do this in 2 commits the first commit has them unchanged and the second one makes your edits so that we can see just the changes on top of BuildTools.